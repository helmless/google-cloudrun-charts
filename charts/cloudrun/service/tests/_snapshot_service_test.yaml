suite: Cloud Run Service > Snapshot Tests
templates:
  - cloudrun.yaml
set:
  type: service
tests:
  - it: should render default manifest without errors
    asserts:
      - matchSnapshot: {}
