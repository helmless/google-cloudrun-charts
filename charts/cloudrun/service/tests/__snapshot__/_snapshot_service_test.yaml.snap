should render default manifest without errors:
  1: |
    apiVersion: serving.knative.dev/v1
    kind: Service
    metadata:
      annotations:
        run.googleapis.com/ingress: all
        run.googleapis.com/launch-stage: GA
      labels: null
      name: serverless-helm
    spec:
      template:
        metadata:
          annotations:
            autoscaling.knative.dev/maxScale: "100"
            run.googleapis.com/cpu-throttling: "true"
            run.googleapis.com/execution-environment: gen2
        spec:
          containerConcurrency: 80
          containers:
            - image: us-docker.pkg.dev/cloudrun/container/hello
              livenessProbe:
                failureThreshold: null
                httpGet:
                  path: /
                  port: 8080
                initialDelaySeconds: null
                periodSeconds: null
              ports:
                - containerPort: 8080
              readinessProbe:
                failureThreshold: null
                httpGet:
                  path: /
                  port: 8080
                initialDelaySeconds: null
                periodSeconds: null
              resources:
                limits:
                  cpu: "1"
                  memory: 512Mi
              startupProbe:
                failureThreshold: null
                httpGet:
                  path: /
                  port: 8080
                initialDelaySeconds: null
                periodSeconds: null
          timeoutSeconds: 300
